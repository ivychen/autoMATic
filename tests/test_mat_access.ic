// Test matrix access
void accessMat(matrix M) {
	int a;

	a = M[1][1];
}
