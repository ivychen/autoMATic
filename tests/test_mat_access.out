
void accessMat(M)
{
int a;
a = M[1][1];
}
